sources/ActionView.cpp

```diff
@@ -29,9 +30,30 @@ ActionView::ActionView(const char* name, BMessage* action, const int32& flags)
 	fValueBox = new AutoTextControl("valuebox", NULL, NULL, new BMessage());
 	fValueBox->SetDivider(0);
 
+	fAddButton = new BButton(kAdd);
+	fRemoveButton = new BButton(kRemove);
+
+	BMessage* msg = new BMessage(MSG_ADD_ACTION);
+	msg->AddPointer(kAdd, this);
+	fAddButton->SetMessage(msg);
+
+	msg = new BMessage(MSG_REMOVE_ACTION);
+	msg->AddPointer(kRemove, this);
+	fRemoveButton->SetMessage(msg);
+
+	BFont font;
+	GetFont(&font);
+	float width = font.StringWidth("+-") * 2;
+	BSize size(width, width);
+
+	fAddButton->SetExplicitSize(size);
+	fRemoveButton->SetExplicitSize(size);
+
 	BLayoutBuilder::Group<>(this, B_HORIZONTAL, B_USE_DEFAULT_SPACING)
 		.Add(fActionField, 0)
 		.Add(fValueBox)
+		.Add(fRemoveButton, 0)
+		.Add(fAddButton, 0)
```

Comment:
Good idea. I use `B_USE_HALF_ITEM_SPACING` to make it tighter:
![git diff](https://user-images.githubusercontent.com/6434360/27506507-08d3207c-586e-11e7-9f91-682a4e463fbb.png)